tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_details.py

```diff
@@ -94,11 +94,9 @@ def a11y_click_edit(self):
         return EditContact(self.marionette)
 
     def tap_back(self):
-        el = self.marionette.find_element(*self._details_header_locator)
-        Wait(self.marionette).until(expected.element_displayed(el))
-        # TODO: remove tap with coordinates after Bug 1061698 is fixed
-        el.tap(25, 25)
-        Wait(self.marionette).until(expected.element_not_displayed(el))
+        back_button = self.marionette.find_element(*self._details_header_locator)
+        self.tap_element_from_system_app(element=back_button)
+        Wait(self.marionette).until(expected.element_not_displayed(back_button))
```

Comment:
tap_element_from_system_app is switching to the system app, so I'm not sure how this works.
In general, I would prefer it we're going away from negative checks like these.
But I guess it works, because otherwise you haven't written it like this.